daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/IdString.scala

```diff
@@ -262,18 +262,18 @@ private[data] final class IdStringImpl extends IdString {
     @throws[IllegalArgumentException]
     override def assertFromString(s: String) = {
       if (s.length == 0)
-        throw new IllegalArgumentException("DAML LF Name is empty")
+        throw new IllegalArgumentException("Daml LF Name is empty")
```

Comment:
```suggestion
        throw new IllegalArgumentException("Daml-LF Name is empty")
```
We probably should use the occasion, to make "Daml LF" and Daml-LF" consistent.

Reply:
Good point -- ran a search & replace to do that.